Exercise_B/index.css

```diff
@@ -2,3 +2,36 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
```

Comment:
I would create a new class that contains the height and width properties that you repeat per id, apply this class in your HTML to the divs. This saves you some code. 

You should be able to remove display flex from line 20, 26 and 35. 
This has already been stated on line 7